BuildingBlocks/src/BuildingBlocks.Infrastructure/EventBus/AzureServiceBus/EventBusAzureServiceBus.cs

```diff
@@ -163,24 +164,92 @@ private async Task<bool> ProcessEvent(string eventName, string message)
             try
             {
                 var policy = EventBusRetryPolicyFactory.Create(
-                    _handlerRetryBehavior,
-                    (ex, _) => _logger.LogWarning(
-                        "The following error was thrown while executing '{eventHandlerType}':\n'{errorMessage}'\n{stacktrace}.\nAttempting to retry...",
-                        eventType.Name,
-                        ex.Message,
-                        ex.StackTrace)
-                    );
+                    _handlerRetryBehavior, (ex, _) => _logger.ErrorWhileExecutingEventHandlerType(eventType.Name, ex.Message, ex.StackTrace!, ex));
 
                 await policy.ExecuteAsync(() => (Task)concreteType.GetMethod("Handle")!.Invoke(handler, new[] { integrationEvent })!);
             }
             catch (Exception ex)
             {
-                _logger.LogError(ex,
-                    $"An error occurred while processing the integration event with id '{integrationEvent.IntegrationEventId}'.");
+                _logger.ErrorWhileProcessingIntegrationEvent(integrationEvent.IntegrationEventId, ex);
                 return false;
             }
         }
 
         return true;
     }
 }
+
+file static class LoggerExtensions
```

Comment:
I would expect all services buses to contain the same log messages (because they operate similarly. Can you check if we forgot some of the log messages?
The messages in this LoggerExtensions class look fine. So it can serve as en example for the others.

We can also have a look at it together.

Reply:
I compared the service buses and document which is missing which methods in comparison to `EventBusAzureServiceBus`, as well as which additional methods each one has. We can take a look together then.